src/components/VmDetails/cards/NicsCard/style.css

```diff
@@ -19,3 +58,35 @@
   font-size: 0.7em;
   padding: 0 10px;
 }
+.ip6-container {
+  padding-left: 0;
+}
+
+.nic-row-actions {
+  display: table-cell;
+  white-space: nowrap;
+  padding-left: 5px;
+}
+
+.action {
```

Comment:
Maybe better to move this class to `index-nomodules`, because it can be used in the future in other components?

Reply:
Moved to `VmDetails/itemListStyle.css` to be shared with Disks card